src/protocol/codec.c

```diff
@@ -296,8 +296,7 @@ int8_t _z_bytes_val_encode(_z_wbuf_t *wbf, const _z_bytes_t *bs) {
     int8_t ret = _Z_RES_OK;
 
     if ((wbf->_expansion_step != 0) && (bs->len > Z_TSID_LENGTH)) {
-        // ret |= _z_wbuf_wrap_bytes(wbf, bs->start, 0, bs->len);
-        ret |= _z_wbuf_write_bytes(wbf, bs->start, 0, bs->len);
+        ret |= _z_wbuf_wrap_bytes(wbf, bs->start, 0, bs->len);
```

Comment:
It seems that this line is being changed frequently. Last time it was from `wrap` to `write` in https://github.com/eclipse-zenoh/zenoh-pico/commit/5238f256f8ccb4f737e82552fb0241f721b8d691 .

Reply:
Yah that's one of the whole point of this PR, reverting this change from @p-avital that outlived its usefulness.